content/en/docs/ambient/architecture/hbone/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: HBONE
+description: Understanding Istio's secure tunneling protocol.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+
+## Understanding the HBONE protocol
+
+HBONE (HTTP Based Overlay Network Encapsulation) is an Istio-specific term. It is a mechanism to transparently multiplex packets related to many different application connections over a single, mTLS encrypted network connection - an encrypted tunnel.
```

Comment:
'packets' or 'TCP streams' ?

similarly in the "... many different application connections..." section. 

No need to pressage QUIC :)

Reply:
yeah `packets` is left over from the previous doc. Agree tho.